docs/UserGuide.md

```diff
@@ -87,87 +100,119 @@ Now that you are familiar with Reache's interface, we recommend visiting [Gettin
 
 ### 3.1 Managing Contacts
 #### 3.1.1 Adding a contact: `add`
-Adds a person to the contact list.
+Add a person to your contact list by specifying details such as:
+- Name
+- Job title
+- Company
+- Tag(s)
+- Pronoun(s)
+- Phone number(s) 
+- Email id(s)
+- Physical address(es)
+
+_Note_: 
+- When adding contacts, 'name' is the only compulsory field. 
+- You can add optional labels to your contact's phone numbers, email ids and physical addresses.
+- Command words for the fields that can be added are given under the [5. Contact Fields Summary](#5-contact-fields-summary) section.
 
 Format: `add n/<NAME> [ph/<PHONE_NUMBER> l/<LABEL>]... [e/<EMAIL> l/<LABEL>]... [a/<ADDRESS> l/LABEL]... [t/TAG]...`
 
-After adding a person to the contact list, you will be navigated to their contact details page. 
-From this page, you can edit their contact details using the commands found under the [Editing Information](#33-editing-information) section.
+Example: `add n/Alex Dunphy t/friend`
+
+<br>
+
+#### 3.1.2 Editing a contact: `edit`
+On the contact details page, modify contact details of the person displayed on the page. The fields that can be added/edited are given in the [5. Contact Fields Summary](#5-contact-fields-summary) section.
```

Comment:
In the intro we use the term Contact Details Window and in the guide we use contact details page. I'm not sure how big an issue you think it is if at all, but I just wanted to make a note.

Reply:
Standardised it across the UG